### PR TITLE
backport from picasa connector

### PR DESCRIPTION
**Pull request by ymkins**

### CODE REVIEW

File: sdk/errors/errors.go [hunk 1]
```diff
@@ -80,3 +82,17 @@ func IsErrorConnectionReset(err error) bool {
 	}
 	return errors.Is(err, syscall.ECONNRESET)
 }
+
+// IsErrorTimedOut verifies error
+func IsErrorTimedOut(err error) bool {
+	if runtime.GOOS == "windows" {
```
Comment (VladislavSenkevich) [inline sdk/errors/errors.go:88]: Better to move "windows" to const

File: sdk/clients/request.go [hunk 2]
```diff
@@ -16,6 +17,7 @@ var httpClient = &http.Client{
 	Transport: &http.Transport{
 		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
 	},
+	Timeout: time.Duration(5 * time.Second),
```
Comment (VladislavSenkevich) [inline sdk/clients/request.go:20]: magic number?

**Review by VladislavSenkevich — CHANGES_REQUESTED** [review]

File: sdk/clients/request.go (hunk 2 above)
Comment (ymkins) [reply sdk/clients/request.go:20]: well, NetClientTimeout provides ability to change 5sec defaults

File: sdk/errors/errors.go (hunk 1 above)
Comment (ymkins) [reply sdk/errors/errors.go:88]: actually, never seen such examples on github)

**Pull request merged**